lib/trento/application/integration/checks/adapter/runner.ex

```diff
@@ -57,7 +57,7 @@ defmodule Trento.Integration.Checks.Runner do
         Enum.map(host_settings, fn host ->
           %{
             host_id: host.host_id,
-            address: host.ssh_address,
+            address: "",
```

Comment:
Setting this empty by now. This complete function will be removed once we remove the connection settings code. I didn't want to do it here, to not overload the PR more

Reply:
could you add a FIXME reminder?

Reply:
The complete file will be gone once we finish removing the legacy runner usage. 
I'm actually already working on the removal in other branches. I don't think we will forget anyhow about this